docs/DeveloperGuide.md

```diff
@@ -342,13 +342,48 @@ retrieving the `ActivityGoalList` from the database and displaying the goals to
 
 5. **Result Display**: A message is returned post-execution and passed through AthletiCLI to the UI for display to the user.
 
+In particular to demonstrate all parser classes, the following sequence diagram shows how the 'edit-sleep' command works:
```

Comment:
```suggestion
In particular to demonstrate all parser classes, the following sequence diagram shows how the `edit-sleep` command works:
```

Reply:
apologies just restored the misc commands